R/pkg/R/context.R

```diff
@@ -319,6 +319,34 @@ spark.addFile <- function(path, recursive = FALSE) {
   invisible(callJMethod(sc, "addFile", suppressWarnings(normalizePath(path)), recursive))
 }
 
+
+#' Adds a JAR dependency for all tasks to be executed on this SparkContext in the future.
+#'
+#' The \code{path} passed can be either a local file, a file in HDFS (or other Hadoop-supported
+#' filesystems), an HTTP, HTTPS or FTP URI, or local:/path for a file on every worker node.
+#' If \code{addToCurrentClassLoader} is true, add the jar to the current threads' classloader. In
+#' general adding to the current threads' class loader will impact all other application threads
+#' unless they have explicitly changed their class loader.
```

Comment:
should
```
add the jar to the current threads' classloader. In
 +#' general adding to the current threads' class loader will impact all other application threads
 +#' unless they have explicitly changed their class loader.
```
be reworded for R or Python? It should be the JVM. `threads` or `classloader` aren't familiar concepts to R users.